ksqldb-serde/src/main/java/io/confluent/ksql/serde/avro/AvroFormat.java

```diff
@@ -81,9 +82,12 @@ protected ConnectSchemaTranslator getConnectSchemaTranslator(
     // Ensure schema is compatible by converting to Avro schema:
     getConnectSchemaTranslator(formatProps).fromConnectSchema(connectSchema);
 
-    final String schemaFullName = new AvroProperties(formatProps).getFullSchemaName();
+    final AvroProperties properties = new AvroProperties(formatProps);
```

Comment:
For JSON_SR and PB we don't have a corresponding `XxxProperties` helper class, and do the schema-id parsing "manually". Would it be worth to introduce such helpers (also to unify the code pattern)?

Reply:
Yeah. That's the plan